composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Sheer/knowlegebase",
+    "name": "Sheer/knowledgebase",
```

Comment:
You can pre-make a backup copy of the table of this extension and then use these data for new renamed extension.